services/static-webserver/client/tools/qooxdoo-kit/builder/Dockerfile

```diff
@@ -4,7 +4,7 @@
 #
 # Note: context at osparc-simcore/services/static-webserver/client expected
 #
-ARG tag
+ARG tag=latest
```

Comment:
Q: I think this is wrong. Why provide a default here since it was required to pass it in from the cli?

Reply:
This fixes a warning follows the instructions

<img width="1173" height="218" alt="image" src="https://github.com/user-attachments/assets/029e4a2f-176b-433d-b0e2-fae4c39d8042" />


SEE https://docs.docker.com/go/dockerfile/rule/invalid-default-arg-in-from/